back-end/src/main/java/box/BoxGetter.java

```diff
@@ -43,7 +43,7 @@
             if (curLayer.isStatic())
                 data.add(new ArrayList<>());
             else
-                data.add(curLayer.getIndexer().getDataFromRegion(c, i, deltaWkt, predicates.get(i)));
+                data.add(curLayer.getIndexer().getDataFromRegion(c, i, deltaWkt, predicates.get(i), newBox, oldBox));
```

Comment:
This is good catch. Xiaoyu ran into the same issue when implementing fetching logic for SciDB.